src/server/parser.ts

```diff
@@ -35,13 +35,13 @@ function startIndex() {
   )
 
   scanProcess.on('message', (mess) => {
-    const { data, log } = mess
+    const { data, msglog } = mess
```

Comment:
This change have conflicts with https://github.com/iamcco/vim-language-server/pull/16

Reply:
I have resolve it